docs/modules/ROOT/pages/logging/tutorial-lokistack.adoc

```diff
@@ -0,0 +1,109 @@
+= Tutorial: Access application logs with Lokistack
+
+This tutorial explains how to view the logs of an application on Openshift.
+
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry
+
+
+== Step 1: Deploy Fortune application
+
+. Create a new namespace called `vshn-demo-fortune`
```

Comment:
I'd prefer to not use a namespace prefixed with `vshn-`. While there's no policies in place to prevent arbitrary users from using the `vshn-` prefix, I don't think it's a particularly good idea to use the prefix in a tutorial.